packages/myst-to-tex/src/math.ts

```diff
@@ -1,5 +1,35 @@
 import type { Handler, ITexSerializer } from './types.js';
 
+// Top level environments in amsmath version 2.1 (and eqnarray), see:
+// http://anorien.csc.warwick.ac.uk/mirrors/CTAN/macros/latex/required/amsmath/amsldoc.pdf
+const ENVIRONMENTS = [
+  'equation',
+  'multline',
+  'gather',
+  'align',
+  'alignat',
+  'flalign',
+  'matrix',
+  'pmatrix',
+  'bmatrix',
+  'Bmatrix',
+  'vmatrix',
+  'Vmatrix',
+  'eqnarray',
+];
+
+const RE_OPEN = new RegExp(`^\\\\begin{(${ENVIRONMENTS.join('|')})([*]?)}`);
+
+function isAmsmathEnvironment(value: string) {
+  const matchOpen = value.trim().match(RE_OPEN);
+  if (!matchOpen) return false;
+  const [, environment, numbered] = matchOpen;
```

Comment:
```suggestion
  const [, environment, not_numbered] = matchOpen;
```

Probably should convert it to proper bool at the end as well. Doesn't really make a difference here, but it might confuse when coming back.

Reply:
Updated to `start` and returning a boolean. Thanks!